src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -1,33 +1,53 @@
 /* ==================================================
- * Authors:
+ * Authors: Lemi Miyu and Fawaaz
  *
  * --------------------------------------------------
- * Description:
- *
+ * Description: Drivetrain.java contains functions
+ * and objects needed with the drivetrain's motors
+ * in order to control robot movement
  * ================================================== */
 
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
 
 public class Drivetrain extends SubsystemBase
 {
   // HARDWARE //
 
   // declare motor controllers, solenoids, and other hardware here
 
-  // CONSTANTS //
+  private PWMSparkMax rightMotor;
+  private PWMSparkMax leftMotor;
```

Comment:
@droid-101 finalized today that the drivetrain will be using 4 Talon SR motor controllers.  So you can use the `Talon` class instead of `PWMSparkMax`.  I believe the electrical team will be using PWM Y-cables to wire up the left and right motor controllers, so from a code perspective, you only need two Talons here:
```
private Talon leftMotors;
private Talon rightMotors;
```
Whatever speed you set to each Talon in the code will automatically be sent to both the Talons on one side of the robot.  This might change though, but for now let's stick with just two and we can separate it later if the electrical team decides not to use PWM Y-cables.